internal/gatewayapi/testdata/httproute-with-header-filter-duplicate-remove-multiple-filters.out.yaml

```diff
@@ -67,10 +67,6 @@ httpRoutes:
       conditions:
       - type: Accepted
         status: "True"
```

Comment:
ditto

Reply:
Commit e14991a updates the PR to only set the "UnsupportedValue" reason when `Accepted=false`.